src/borg/testsuite/archive.py

```diff
@@ -96,6 +96,11 @@ def test_stats_progress_json(stats):
     assert "original_size" not in result
     assert "nfiles" not in result
 
+def test_hashing_time(stats):
+    assert stats.hashing_time >= 0.0
+
+def test_chunking_time(stats):
+    assert stats.chunking_time >= 0.0
```

Comment:
guess this is not really testing something useful yet.

the `stats` you have as argument here comes from the fixture in line 19+.

Reply:
Should I remove these? https://github.com/borgbackup/borg/pull/6991/commits/3ef8052ff8486d09ac8b4cac38399d161b04f65c Replaces `test_hashing_time`, I'm gonna add one for chunking time too.

Reply:
yes, remove